doc/index.md

```diff
@@ -1019,6 +1105,98 @@ Value of event:
     }
     ```
 
+#### `conversationUpdate`
+Emits when softphone conversations change. Some changes that trigger this event
+are, but not limited to:
+* new conversation started
+* conversation ended
+* participants state changed (mute, held, connection state, etc)
+
+> Note: this event will only be available for softphone conversations. Video
+conversations have their own session level events ([see video event docs](video.md#video-session-level-events)).
+
+Declaration:
+``` ts
+sdk.on('conversationUpdate', (event: ISdkConversationUpdateEvent) => { });
+```
+
+Value of event:
+* `event: ISdkConversationUpdateEvent`
+
+``` ts
+interface ISdkConversationUpdateEvent {
+  /**
+   * assumed conversationId of the activce conversation
+   */
+  activeConversationId: string;
+  /**
+   * All current softphone conversations
+   */
+  current: IStoredConversationState[];
+  /**
+   * Newly added softphone conversations
+   */
+  added: IStoredConversationState[];
+  /**
+   * Removed softphone conversations
+   */
+  removed: IStoredConversationState[];
+}
+
+interface IStoredConversationState {
+  /**
+   * Most recent conversation event received for this conversation
+   */
+  conversationUpdate: ConversationUpdate;
+  /**
+   * conversationId of this conversation
+   */
+  conversationId: string;
+  /**
+   * Webrtc session this conversation is using
+   */
+  session?: IExtendedMediaSession;
+  /**
+   * Most recent participant for the authenticated user
+   */
+  mostRecentUserParticipant?: IConversationParticipantFromEvent;
+  /**
+   * Most recent call start for the authenticated user
+   */
+  mostRecentCallState?: ICallStateFromParticipant;
+}
+```
+
+A few things to note about this event:
+1. `ISdkConversationUpdateEvent.activeConversationId` is not guaranteed to be accurate and sometimes it will be `''`.
+  This is rare, but usually this is seen when there are mutliple active calls all on hold.
+  As soon as one is taken off hold, this event will then be able to determine the correct active conversation ID.
+1. `IStoredConversationState.session` can be `null` because there are times when the conversation will not have a session
```

Comment:
NIT: these are all numbered 1.

Reply:
Markdown will return these an an ordered list correctly. This way we don't have to manage the bullet point numbers ourselves. 

![image](https://user-images.githubusercontent.com/32391370/145472892-3d04c4bf-d4d9-40a1-8012-fcf050f26379.png)